docs/src/language_servers.md

```diff
@@ -751,6 +752,33 @@ See [Javascript/TypeScript](#javascripttypescript).
     }
     ```
 
+5. Optional: to enable some useful commands provided by language server, add the following to `*.sublime-commands` (see more [here](commands.md#execute-server-commands)):
+
+    <!-- how to call: see https://github.com/nvarner/typst-lsp/blob/master/editors/vscode/src/extension.ts -->
+    ```jsonc title="Packages/User/Default.sublime-commands"
+    [
+        // ...
+        {
+            "caption": "typst-lsp - Pin the main file to the currently openning document",
```

Comment:
```suggestion
            "caption": "typst-lsp - Pin the main file to the currently opened document",
```

?

Reply:
Hmm, I just copied this from original extension :smile: